### PR TITLE
fix: table checkbox for required params on import

### PR DESCRIPTION
**Pull request by marclave**
this pr https://github.com/scalar/scalar/pull/2469 unfortunately introduced a regression for the api client standalone app, now we just set enabled on import :)

### CODE REVIEW

File: packages/api-client/src/store/workspace.ts [hunk 1]
```diff
@@ -197,6 +197,7 @@ const createParamInstance = (param: OpenAPIV3_1.ParameterObject) =>
           : '',
     description: param.description,
     required: param.required,
+    enabled: !!param.required,
```
Comment (amritk) [inline packages/api-client/src/store/workspace.ts:200]:
```suggestion
    enabled: Boolean(param.required),
```
nit pick but this one is a bit more succinct

Comment (amritk) [reply packages/api-client/src/store/workspace.ts:200]: but also shouldn't even be needed?

**Review by amritk — APPROVED** [review]
Nit pick comment/question

File: packages/api-client/src/store/workspace.ts (hunk 1 above)
Comment (marclave) [reply packages/api-client/src/store/workspace.ts:200]: woah ive never seen this before 🤯 yeah i think youre right, lemme remove

Comment (marclave) [reply packages/api-client/src/store/workspace.ts:200]: well, it could be undefined 
image

**Pull request merged**